src/snowcli/cli/object/show/commands.py

```diff
@@ -4,23 +4,23 @@
 
 from snowcli.cli.common.decorators import global_options_with_connection
 from snowcli.cli.common.flags import DEFAULT_CONTEXT_SETTINGS
-from snowcli.cli.warehouse.manager import WarehouseManager
+from snowcli.cli.object.show.manager import ShowManager
 from snowcli.output.decorators import with_output
 from snowcli.output.types import QueryResult
 
 app = typer.Typer(
-    name="warehouse",
+    name="show",
     context_settings=DEFAULT_CONTEXT_SETTINGS,
     help="Manages warehouses.",
```

Comment:
```suggestion
    help="List existing objects of given type",
```
Also we should make it a command with single argument `[TYPE]` and `--like` flag.